csi/src/server.rs

```diff
@@ -175,6 +183,17 @@ async fn main() -> Result<(), String> {
     }
     builder.init();
 
+    if let Some(nvme_io_timeout_secs) = matches.value_of("nvme_core io_timeout")
+    {
+        let io_timeout_secs: u32 = nvme_io_timeout_secs
+            .parse()
+            .expect("nvme_core io_timeout should be an integer number, representing the timeout in seconds");
+
+        if let Err(error) = dev::nvmf::set_nvmecore_iotimeout(io_timeout_secs) {
+            error!("Failed to set nvme_core io_timeout: {}", error.to_string());
```

Comment:
Shouldn't we exit with an error here? Not honouring io timeout setting can have bad consequences and message in the log file can be easily overlooked.

Reply:
Ok, changed it to a panic.